### PR TITLE
Demo by Ash

### PR DESCRIPTION
**Pull request by ashtilawat23**
What work was done?

Why was this work done?

Future work, if applicable:

Loom video []

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Basic SPA
 
-For steps on how to work with this repository [please see here](https://docs.labs.lambdaschool.com/labs-spa-starter/)
+For the steps on how to work with this repository [please see here](https://docs.labs.lambdaschool.com/labs-spa-starter/)
```
Comment (stevenjhomem) [inline README.md:4]: Ash you are doing great.... ;)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
```
Comment (zach-morris-txt) [inline README.md:3]: Excellent

Comment (ehsan2793) [inline README.md:3]: oh you added the

**Review by zach-morris-txt — APPROVED** [review]
Approve

**Pull request merged**